handshake/interface.go

```diff
@@ -0,0 +1,19 @@
+package handshake
+
+import "github.com/lucas-clemente/quic-go/protocol"
+
+// Sealer seals a packet
+type Sealer func(dst, src []byte, packetNumber protocol.PacketNumber, associatedData []byte) []byte
```

Comment:
Actually, we could even split up crypto.AEAD into a Sealer and a Opener and reuse the Sealer interface from there (or embed it in a handshake.Sealer).